customhooksreact/src/hooks/useInput.ts

```diff
@@ -0,0 +1,16 @@
+import { useState } from 'react';
+
+const useInput = (initialValue: string) => {
```

Comment:
use interface for return type

Reply:
used interface for return type, resolved.